src/BigQueryFunctions/DeleteTable.ts

```diff
@@ -0,0 +1,11 @@
+//Deletes given table in given database 
+export default async function deleteTable(datasetId: string, tableId: string): Promise<void> {
```

Comment:
Please have all the big query-related functions like create, delete, insertion or table exists check, etc inside one class. Have this as a service class. Using that service class, one should be able to do any of these operations.